src/sourmash/distance_utils.py

```diff
@@ -0,0 +1,277 @@
+"""
+Utility functions for jaccard/containment --> distance estimates
+From https://github.com/KoslickiLab/mutation-rate-ci-calculator
+https://doi.org/10.1101/2022.01.11.475870
+"""
+from scipy.optimize import brentq
+from scipy.stats import norm as scipy_norm
+from numpy import sqrt
+from math import log, exp
+
+from .logging import notify
+
+
+def r1_to_q(k, r1):
+    r1 = float(r1)
+    q = 1 - (1 - r1) ** k
+    return float(q)
+
+
+def var_n_mutated(L, k, r1, q=None):
+    # there are computational issues in the variance formula that we solve here
+    # by the use of higher-precision arithmetic; the problem occurs when r is
+    # very small; for example, with L=10,k=2,r1=1e-6 standard precision
+    # gives varN<0 which is nonsense; by using the mpf type, we get the correct
+    # answer which is about 0.000038.
+    if r1 == 0:
+        return 0.0
```

Comment:
add test?

Reply:
added in 01d4ae742408d9abf6684b3de747966ece212e22